synapseclient/client.py

```diff
@@ -811,7 +813,8 @@ def _resolve_download_path_collisions(self, downloadLocation, file_name, ifcolli
                                  'for "ifcollision"' % ifcollision)
         return downloadPath
 
-    def store(self, obj, **kwargs):
+    def store(self, obj, *, createOrUpdate=True, forceVersion=True, versionLabel=None, isRestricted=False,
```

Comment:
Not an issue w/ this PR, but the casing inconsistency in some of the public functions especially is not ideal. This just unrolls the current kwargs so no change but if we did want to clean this up at some point without immediate breaking changes some kwargs to catch old style params and convert them could be usefu.